automatic/graphviz/README.tmp

```diff
@@ -1,9 +1,13 @@
-# <img src="https://cdn.jsdelivr.net/gh/chocolatey/chocolatey-coreteampackages@00f392142cdbdbda147d3cc3ccb1cb593afb996d/icons/graphviz.png" width="48" height="48"/> [graphviz](https://chocolatey.org/packages/graphviz)
-
-Graph visualization is a way of representing structural information as diagrams of abstract graphs and networks. It has important applications in networking, bioinformatics, software engineering, database and web design, machine learning, and in visual interfaces for other technical domains.
-
-Graphviz is open source graph visualization software. It has several main layout programs. See the gallery for sample layouts. It also has web and interactive graphical interfaces, and auxiliary tools, libraries, and language bindings. We're not able to put a lot of work into GUI editors but there are quite a few external projects and even commercial tools that incorporate Graphviz. You can find some of these in the Resources section.
-
-The Graphviz layout programs take descriptions of graphs in a simple text language, and make diagrams in useful formats, such as images and SVG for web pages; PDF or Postscript for inclusion in other documents; or display in an interactive graph browser.
-
-Graphviz has many useful features for concrete diagrams, such as options for colors, fonts, tabular node layouts, line styles, hyperlinks, and custom shapes.
+# [<img src="https://cdn.jsdelivr.net/gh/chocolatey/chocolatey-coreteampackages@00f392142cdbdbda147d3cc3ccb1cb593afb996d/icons/graphviz.png" width="48" height="48"/> <PackageName>](https://chocolatey.org/packages/<PackageName>)
+
+** Note **
+
+Starting with version _2.44_ this package contains cmake builds of graphwiz. If you encounter problem with missing executables for the time being continue using the last msbuild version _2.38_

```

Comment:
I guess it's too late now, but there's a typo here: s/graphwiz/Graphviz

Reply:
Fixed. Thx.

Reply:
While you're in this area, the grammar of the second sentence could be improved with s/encounter problem/encounter problems, and adding a trailing period.

Reply:
OK, done, in next release visible tho.

Reply:
Thanks!